app/services/notices/setup/fetch-returns-recipients.service.js

```diff
@@ -183,20 +185,27 @@ async function _fetchRecipient(session) {
 
 async function _fetchRecipients(session) {
   const {
-    determinedReturnsPeriod: { dueDate, summer },
+    determinedReturnsPeriod: { dueDate, endDate, startDate, summer, quarterly },
     removeLicences = ''
   } = session
 
   const excludeLicences = transformStringOfLicencesToArray(removeLicences)
 
+  const bindings = [endDate, startDate, summer, quarterly, excludeLicences, excludeLicences, excludeLicences]
+
   const where = `
-    AND rl.due_date = ?
-    AND rl.metadata->>'isSummer' = ?
-  `
+      AND rl.due_date ${featureFlagsConfig.enableNullDueDate ? 'IS NULL' : '?'}
```

Comment:
I don't see the tests updated to check both ? 

Only the seeder updated to use the start date  / end date instead of end date. 

might be one to discuss

Reply:
@robertparkinson is right. I've previously said, unless we're months away from shipping a feature (we're not on dynamic due dates), go ahead and write/update the unit tests to believe that is the only state the world will be in.

Reply:
Okay. 

https://github.com/DEFRA/water-abstraction-system/pull/2548 is ready. 

Either way of merging works i think.